roles/control_center/defaults/main.yml

```diff
@@ -37,7 +37,7 @@ control_center_service_overrides:
 control_center_service_environment_overrides:
   ROCKSDB_SHAREDLIB_DIR: "{{control_center_rocksdb_path}}"
   CONTROL_CENTER_HEAP_OPTS: "-Xmx6g"
-  CONTROL_CENTER_OPTS: "{{ control_center_final_java_args | java_arg_build_out }}"
+  CONTROL_CENTER_OPTS: "{{ control_center_final_java_args | confluent.platform.java_arg_build_out }}"
   CONTROL_CENTER_LOG4J_OPTS: "{% if control_center_custom_log4j|bool %}-Dlog4j.configuration=file:{{control_center.log4j_file}}{% endif %}"
   # Remove trailing slash if there is one
   LOG_DIR: "{{ control_center_log_dir|regex_replace('\\/$', '') }}"
```

Comment:
Random question, but do the out of the box filters need the FQFN? I'm guessing no, or at least its not needed for this PR.

I'm wondering if we want to move everything that direction at some point.

Reply:
No, FQN is not required yet. But all docs, examples etc. use FQN. And Ansible might require it one day.... We should also do it, but that will be a massive change.... :-)